packages/web-components/src/global/variables.css

```diff
@@ -31,6 +31,7 @@
   --color-secondary-dark: #cd2026;
   --color-secondary-light: #e59393;
   --color-secondary-lightest: #f9dede;
+  --v3-color-error-dark: #b50909;
```

Comment:
Everything looks good to me! Do you think we still need this custom property here though now that we are pulling the error color directly from the css-library variables file?

Reply:
I just added it there to keep it consistent and to have it available in case we need it in a `.css` file, in case we can't change it to a `.scss` file.